docs/source/documentation/concepts.rst

```diff
@@ -177,3 +177,43 @@ There are two ways to run several tasks in parallel on a same organization. The
 .. TODO:: Detail vertical and horizontal scaling
 
 .. TODO:: Explain what is substra tools
+
+
+Compute plan execution
+-----------------------
+
+Read this paragraph to understand what happens during the compute plan execution and understand what can be done to improve the execution time.
+
+Once a compute plan is submitted to the platform, its tasks are scheduled to be executed on each organisation.
```

Comment:
```suggestion
Once a compute plan is submitted to the platform, its tasks are scheduled to be executed on each organization.
```
I think we use the American version everywhere else.

Reply:
yes, thanks !